### PR TITLE
SFD-5602 Enable nhs choices in fuzzy

### PR DESCRIPTION
**Pull request by sethaguda**

### CODE REVIEW

File: build/automation/var/profile/local.mk [hunk 1]
```diff
@@ -47,3 +47,8 @@ POSTCODE_MAPPING_PASSWORD := password
 GOOGLE_API_URL := https://maps.google.com/maps/api
 GOOGLE_API_ADDRESS_URI := /geocode/json
 GOOGLE_MAPS_API_KEY		   := gibarish
+
+#NHS Choices
+NHS_CHOICES_API_URL := https://api.nhs.uk
+#Trial NHS_CHOICES_API_KEY
+NHS_CHOICES_API_KEY := 7cc220dc7d134f8c906c2f7a1782a61fm
```
Comment (daonhs) [inline build/automation/var/profile/local.mk:54]: Please replace with a dummy key, this repo is not private

File: build/automation/var/profile/pd.mk [hunk 2]
```diff
@@ -146,3 +146,8 @@ TF_VAR_ip_reputation_list_metric_name = $(SERVICE_PREFIX)-waf-aws-ip-reputation-
 TF_VAR_common_rule_set_metric_name = $(SERVICE_PREFIX)-waf-aws-common-rule-set-metric
 TF_VAR_sql_injection_rules_metric = $(SERVICE_PREFIX)-waf-aws-bad-inputs-rule-set-metric
 TF_VAR_bad_input_metric_name = $(SERVICE_PREFIX)-waf-aws-bad-inputs-rule-set-metric
+
+
+#NHS Choices
+NHS_CHOICES_API_URL := https://api.nhs.uk
+NHS_CHOICES_API_URL := MOCK_NHS_API_KEY
```
Comment (daonhs) [inline build/automation/var/profile/pd.mk:153]: NHS_CHOICES_API_URL -> NHS_CHOICES_API_KEY

**Review by daonhs — CHANGES_REQUESTED** [review]

**Review by daonhs — DISMISSED** [review]
Please review comments, theres a few sanity changes that needs to happen

File: application/app/src/main/java/uk/nhs/digital/uec/api/config/ConcurrentSearchConfig.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+import org.springframework.context.annotation.Configuration;
+import org.springframework.scheduling.annotation.EnableAsync;
+
+import java.util.concurrent.*;
+
+@Configuration
+@EnableAsync
+public class ConcurrentSearchConfig {
+
+/*  @Bean
```
[the remark this answers is not in the document]
Comment (sethaguda) [reply application/app/src/main/java/uk/nhs/digital/uec/api/config/ConcurrentSearchConfig.java]: deleted

File: build/automation/var/profile/pd.mk (hunk 2 above)
Comment (sethaguda) [reply build/automation/var/profile/pd.mk]: changed

File: application/app/src/main/java/uk/nhs/digital/uec/api/model/nhschoices/KeyValue.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package uk.nhs.digital.uec.api.model.nhschoices;
+
+public class KeyValue {
```
[the remark this answers is not in the document]
Comment (sethaguda) [reply application/app/src/main/java/uk/nhs/digital/uec/api/model/nhschoices/KeyValue.java:3]: deleted

File: application/app/src/main/java/uk/nhs/digital/uec/api/model/nhschoices/Metric.java [hunk 5]
```diff
@@ -0,0 +1,9 @@
+package uk.nhs.digital.uec.api.model.nhschoices;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class Metric {
```
[the remark this answers is not in the document]
Comment (sethaguda) [reply application/app/src/main/java/uk/nhs/digital/uec/api/model/nhschoices/Metric.java:8]: deleted

File: application/app/src/main/java/uk/nhs/digital/uec/api/model/nhschoices/AgeRange.java [hunk 6]
```diff
@@ -0,0 +1,4 @@
+package uk.nhs.digital.uec.api.model.nhschoices;
+
+public class AgeRange {
```
[the remark this answers is not in the document]
Comment (sethaguda) [reply application/app/src/main/java/uk/nhs/digital/uec/api/model/nhschoices/AgeRange.java:3]: deleted

File: application/app/pom.xml [hunk 7]
```diff
@@ -231,7 +245,7 @@
                     <limit>
                       <counter>METHOD</counter>
                       <value>COVEREDRATIO</value>
-                      <minimum>70%</minimum>
+                      <minimum>60%</minimum>
```
[the remark this answers is not in the document]
Comment (sethaguda) [reply application/app/pom.xml]: refactored

File: application/app/src/main/java/uk/nhs/digital/uec/api/model/ApiRequestParams.java [hunk 8]
```diff
@@ -10,7 +10,7 @@
 import org.springframework.stereotype.Component;
 
 @Component
-@Scope(value = "request", proxyMode = ScopedProxyMode.TARGET_CLASS)
+//@Scope(value = "request", proxyMode = ScopedProxyMode.TARGET_CLASS)
```
[the remark this answers is not in the document]
Comment (sethaguda) [reply application/app/src/main/java/uk/nhs/digital/uec/api/model/ApiRequestParams.java:13]: removed

**Pull request merged**